content/blog/2023/when-to-choose-a-responsive-web-app.md

```diff
@@ -0,0 +1,160 @@
+  Let's start with the responsive web app (RWA).
+---
+{% import 'embed.macros.njk' as embed %}
+
+You have an idea for a custom digital product you’d like to build, and it’s time to decide how to build it. One of your friends swears you won’t succeed unless you build a native mobile app for iPhone. A colleague argues that a responsive web app is the only way to go. Who’s right? Are there alternative options? The decision of how to build your app is quite important – affecting project scope, timeline, and budget – but understanding when to choose one option over another can be overwhelming.
+
+The great news is that you don’t need to spend years becoming a web developer yourself in order to make a wise choice. In the following three-part series, we’ll dig into the three main options for building mobile apps: Responsive Web Apps, Progressive Web Apps, and Native Mobile Apps, and how to determine the best option for your project. As you read, think about who your app will serve, what their most pressing needs are, and how your app will meet those needs.
+
+**Read all three parts to compare app types:**
+
+
+1. [When to Choose a Responsive Web App](/2023/05/05/when-to-choose-a-responsive-web-app/)
+2. [When to Choose a Native Mobile App](/2023/06/07/when-to-choose-a-native-mobile-app/)
+3. [When to Choose a Progressive Web App](/2023/07/05/when-to-choose-a-progressive-web-app/)
+
+
+## What are Responsive Web Apps, PWAs, and Native Mobile Apps?
+Here are the biggest differences in a nutshell.
+
+- **Responsive Web App (RWA)** - An application that runs in a browser, such as Chrome, Firefox, or Safari, and responds to fit any screen size on any device.
+- **Progressive Web App (PWA)** - A responsive web app with additional capabilities that mimic a native mobile app.
+- **Native Mobile App** - An app built specifically for one particular mobile operating system, like iOS or Android.
+
+
+{{ embed.img(
+  src='blog/2023/comparison-chart.jpg',
+  alt='A comparison chart showing the differences between
+  responsive web apps, progressive web apps, and native
+  mobile apps'
+) }}
+
+
+## Responsive Web Apps - Best for Wide Reach
+
+> **TL;DR**
+>
+> If your top priority is to reach as wide a range of
+> people as possible, as quickly as possible, while keeping
+> development and maintenance costs down, a responsive web app is
+> likely the best option for building your digital product.
+
+An RWA is essentially a website that resizes and rearranges its content – responding to fit whatever screen size a person chooses to use. RWAs are built using standard web technologies like HTML, CSS, and JavaScript. They are accessible from any device with a web browser such as Chrome, Firefox, Microsoft Edge, Safari, or a mobile browser.
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2023/yes.png',
+    alt: 'The word yes with a smiley and sparkles'
+  }],
+  attrs={
+    'style': '--column: content-start'
+  }
+) }}
+
+## Choose an RWA when…
```

Comment:
Might be worth mentioning that an RWA can easily _become_ a PWA. So it can be a good first step, when you plan to add more functionality later.